pkg/mcp/server/server.go

```diff
@@ -190,9 +290,13 @@ func (s *Server) newConnection(stream mcp.AggregatedMeshConfigService_StreamAggr
 
 	var types []string
 	for _, typeURL := range s.supportedTypes {
-		con.watches[typeURL] = &watch{
-			responseChan: make(chan *WatchResponse, responseChanDepth),
+		t := time.NewTimer(0)
```

Comment:
Comment on why the value is 0? Presumably because it is to start with an immediate trigger?